carapace-server/src/main/resources/conf/server.dynamic.properties

```diff
@@ -54,7 +54,8 @@ route.100.action=cache-if-possible
 default.action.notfound=not-found
 default.action.internalerror=internal-error
 
-
+# Custom X-Proxy-Path header
+mapper.debug=true
```

Comment:
i think we should set this to false by default, even if this is just an example file